### PR TITLE
Adapter for Quadrat Protocol, powered by 0xPlasma Labs

### PR DESCRIPTION
**Pull request by ilyamk**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/quadrat exports TVL: 
        
 
 
    polygon                   14.27 k
    ethereum                  290.88 k
    optimism                  19.508672118465647
    arbitrum                  0
    
    total                    305.18 k

File: projects/quadrat/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const GRAPH_QUERY = gql`
+    query strategies ($timestamp: Int! $skip: Int! $first: Int!) {
+        strategies (skip:$skip first:$first) {
+            token0 {
+                id
+            }
+            token1 {
+                id
+            }
+            daysData (orderBy: createdAt orderDirection: desc skip:0 first: 2,where: {createdAt_lte:$timestamp}) {
```
Comment (g1nt0ki) [inline projects/quadrat/index.js:22]: Hi, the PR is mostly good except this part, if the graph stops updating/indexing new blocks, we wll be showing stale tvl right? can you add a `createdAt_gte` as well, like two days before current timestamp.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/quadrat exports TVL: 
        
 
 
    ethereum                  176.33 k
    arbitrum                  0
    optimism                  21.215002557262665
    polygon                   10.09 k
    
    total                    186.44 k

**Comment by llamatester** [comment]
The adapter at projects/quadrat exports TVL: 
        
 
 
    ethereum                  175.99 k
    arbitrum                  0
    optimism                  21.44442115408293
    polygon                   10.22 k
    
    total                    186.23 k

File: projects/quadrat/index.js (hunk 1 above)
Comment (g1nt0ki) [reply projects/quadrat/index.js]: @ilyamk this issue still exists right?

**Comment by llamatester** [comment]
The adapter at projects/quadrat exports TVL: 
        
 
 
    ethereum                  0
    optimism                  0
    polygon                   0
    arbitrum                  0
    
    total                    0

File: projects/quadrat/index.js (hunk 1 above)
Comment (ilyamk) [reply projects/quadrat/index.js]: @g1nt0ki it should be fixed in PR #3

**Comment by g1nt0ki** [comment]
I have made a minor fix here, hope I have not introduced a bug: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5084

**Pull request merged**